packages/mui-material/src/Stack/Stack.js

```diff
@@ -44,6 +44,7 @@ const getSideFromDirection = (direction) => {
 export const style = ({ ownerState, theme }) => {
   let styles = {
     display: 'flex',
+    flexDirection: typeof ownerState.direction === 'string' ? ownerState.direction : 'column',
```

Comment:
Would this work? It seems to be what's missing, I assume the `typeof ownerState.direction === 'string' ? ownerState.direction` logic duplicates with the one in handleBreakpoints?

```suggestion
    flexDirection: 'column',
```

Reply:
I can confirm that you are right. `flexDirection: 'column'` seems enough.